.env

```diff
@@ -14,31 +14,47 @@
 # Run "composer dump-env prod" to compile .env files for production use (requires symfony/flex >=1.2).
 # https://symfony.com/doc/current/best_practices.html#use-environment-variables-for-infrastructure-configuration
 
-###> symfony/webapp-pack ###
-MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
-###< symfony/webapp-pack ###
-
 ###> symfony/framework-bundle ###
 APP_ENV=dev
-APP_SECRET=1e5342d4dce5dd1f686d55d943522dfc
+APP_SECRET=187b0d05c80b4ad52baf61c728a90548
```

Comment:
better to write here "!ChangeMe!" It is kind of a convention, to don't forget it on prod

<img width="221" alt="image" src="https://user-images.githubusercontent.com/20707343/213379813-76712ecc-84f1-4ffc-a107-bc72cadc8d02.png">

Reply:
did it